app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -156,14 +157,16 @@ function initializeCommentDropdown() {
   }
 
   const commentsContainer = document.getElementById('comment-trees-container');
+  const articleShowMoreButton = document.getElementById(
+    'article-show-more-button',
+  );
```

Comment:
You could combine your two `getElementById` queries into one `querySelectorAll`, which means you don't have to combine them later too.

```js
const dropDownElements = document.querySelectorAll('#comment-trees-container, #article-show-more-button');

for (const element of dropDownElements ) {
  ...
}
```

Reply:
Gah, my review was just too late 😅

Reply:
Thanks for the review @Link2Twenty! I was being explicit to separate them, but you are right. It could use `querySelectorAll`. I'm just going to let the dust settle in the fix, but if you would like to, feel free to pop up a PR for that change.😎